setup.py

```diff
@@ -37,10 +37,23 @@
 LICENSE = metadata.get('license', 'unknown')
 URL = metadata.get('url', 'http://astropy.org')
 
-# Get the long description from the package's docstring
-__import__(PACKAGENAME)
-package = sys.modules[PACKAGENAME]
-LONG_DESCRIPTION = package.__doc__
+# order of priority for long_description:
+#   (1) set in setup.cfg,
+#   (2) load README.rst,
+#   (3) package docstring
+_cfg_long_description = metadata.get('long_description', '')
+if _cfg_long_description:
+    LONG_DESCRIPTION = _cfg_long_description
+
+elif os.path.exists('README.rst'):
```

Comment:
I'm fine with that -- I tend to switch back and forth. The one benefit to .rst is that they are rendered nicely if submitted to pypi. I only used .rst because we assume that for a README in `scripts/`:
https://github.com/astropy/package-template/blob/master/setup.py#L67